packages/flutter/lib/src/material/slider.dart

```diff
@@ -150,6 +208,38 @@ class _RenderSlider extends RenderConstrainedBox {
     if (newValue == _value)
       return;
     _value = newValue;
+    if (divisions != null)
+      _position.animateTo(newValue, curve: Curves.ease);
+    else
+      _position.value = newValue;
+  }
+
+  int get divisions => _divisions;
+  int _divisions;
+  void set divisions(int newDivisions) {
+    if (newDivisions == _divisions)
+      return;
+    _divisions = newDivisions;
+    markNeedsPaint();
+  }
+
+  String get label => _label;
+  String _label;
+  void set label(String newLabel) {
+    if (newLabel == _label)
+      return;
+    _label = newLabel;
+    additionalConstraints = _getAdditionalConstraints(_label);
+    if (newLabel != null) {
+      _labelPainter
+        ..text = new TextSpan(
+          style: Typography.white.body1.copyWith(fontSize: 10.0),
```

Comment:
I think that someday we're going to need values like this to bottom out in the Theme because custom themes.